javascript/with-react/otelcol-config.yaml

```
@@ -0,0 +1,25 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+        cors:
+          allowed_origins:
+            - "http://*"
+            - "https://*"
+
+processors:
+  batch:
+
+exporters:
+  otlp:
+    endpoint: "https://otlp.last9.io:4317"
```

```suggestion
    endpoint: "otlp.last9.io:443"
    headers:
      "Authorization": "<Authorization Header>"
      
```

@chtushar This is already mentioned here https://arc.net/l/quote/nayqrxbj